python/desc/sims/GCRCatSimInterface/TwinklesClasses.py

```diff
@@ -0,0 +1,202 @@
+import os
+import numpy as np
+from desc.twinkles.twinklesVariabilityMixins import VariabilityTwinkles
+from desc.twinkles import sprinklerCompound
+from lsst.sims.catUtils.exampleCatalogDefinitions.phoSimCatalogExamples import PhoSimSpecMap as psmp
+from lsst.sims.catalogs.db import CompoundCatalogDBObject
+from desc.twinkles import TwinklesCompoundInstanceCatalog, sprinkler
+from . import CompoundDESCQAInstanceCatalog, GalaxyCompoundDESCQAObject, PhoSimDESCQA, PhoSimDESCQA_AGN
+
+
+twinkles_sn_sed_dir = 'spectra_files'
+twinkles_spec_map = psmp
+twinkles_spec_map.subdir_map['(^specFile_)'] = twinkles_sn_sed_dir
+
+_twinkles_defs_file = os.path.join(os.environ['TWINKLES_DIR'], 'data', 'dc2_defs.csv')
+
+_agn_cache_file = os.path.join(os.environ['TWINKLES_DIR'], 'data', 'dc2_agn_cache.csv')
+_sne_cache_file = os.path.join(os.environ['TWINKLES_DIR'], 'data', 'dc2_sne_cache.csv')
+global_agn_params_db = os.path.join('/global/cscratch1/sd/brycek/',
+                                    'proto_dc2_agn', 'test_agn.db')
+
+assert os.path.exists(global_agn_params_db)
```

Comment:
Again: I think we should replace this with `if` block that raise exceptions explicitly telling the user that the relevant file doesn't exist